src/Compilers/CSharp/Test/Semantic/Semantics/UninitializedNonNullableFieldTests.cs

```diff
@@ -516,6 +516,14 @@ class C<T> where T : struct
 
             // [NonNullTypes(false)]
             comp = CreateCompilation(new[] { source, NonNullTypesFalse, NonNullTypesAttributesDefinition }, parseOptions: TestOptions.Regular8);
+
+            // PROTOTYPE(NullableReferenceTypes): It feels like the following warning is confusing 
```

Comment:
>It feels like the following warning is confusing  [](start = 50, length = 49)

Isn't that warning just wrong? `T?` is a nullable reference type, so `F3` can be left initialized, no?